.bazelversion

```diff
@@ -1 +1 @@
-bazelbuild/3.3.0
+bazelbuild/4.2.4
```

Comment:
Is there a particular reason why we're using 4.2.4 instead of something newer?

Reply:
It's the latest version that doesn't introduce breaking changes to this project.